uaclient/cli.py

```diff
@@ -325,10 +313,26 @@ def action_attach(args, cfg):
 
 
 def get_parser():
+    service_line_tmpl = " - {name}: {description}{url}"
+    description = [__doc__]
+    for name, ent_cls in entitlements.ENTITLEMENT_CLASS_BY_NAME.items():
+        if ent_cls.help_doc_url:
+            url = " ({})".format(ent_cls.help_doc_url)
+        else:
+            url = ""
+        service_line = service_line_tmpl.format(
+            name=name, description=ent_cls.description, url=url
+        )
+        if len(service_line) <= 80:
+            description.append(service_line)
+        else:
+            parts = service_line.rsplit(" ", 1)
```

Comment:
This won't guarantee that we're under 80 characters. For example:

```
123456789 123456789 123456789 123456789 123456789 123456789 123456789 123456789 more than one word
```

Is this something we need to worry about?

Reply:
Generally I think not because people designing this looked specifically at line size/length when defining these names. I think if we grow a customized 'description' from ua-contracts response we should add this feature. I'll leave a TODO() in the code here in case we source/override description based on ua-contracts response.

Reply:
on second thought. Let's just guard against the condition as future me won't remember to go back to this part of the code if ua-contracts changes descriptions or URL length etc.
Added a fix.